### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brad2412**
Just a rough draft for the final readme.  Still need to add some links at the bottom and looking for inputs on adding or removing some things.  Or tweaking design, whatever sounds good to everyone!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,114 @@
-# Little Esty Shop
+# LittleEstyShop
 
-## Background and Description
+Welcome to LittleEstyShop! This project is a web application that serves as a platform for merchants to manage their items and invoices. It also provides admins with tools to monitor merchant performance and handle invoices. The application consumes the Unsplash API to display images, including the app logo, item images, merchant images, and random photos on the merchant's dashboard.
```
Comment (algibson1) [inline README.md:3]: Aren't the merchant images & random photos on merchant's dashboard the same thing?

File: README.md [hunk 2]
```diff
@@ -1,22 +1,114 @@
...
-"Little Esty Shop" is a group project that requires students to build a fictitious e-commerce platform where merchants and admins can manage inventory and fulfill customer invoices.
+## Work Completed
+The project implements various features for merchants and admins, as well as API consumption for images. Here is a summary of the work completed:
 
-## Learning Goals
-- Practice designing a normalized database schema and defining model relationships
-- Utilize advanced routing techniques including namespacing to organize and group like functionality together.
-- Utilize advanced active record techniques to perform complex database queries
-- Practice consuming a public API while utilizing POROs as a way to apply OOP principles to organize code
+## Merchant Dashboard
```
Comment (algibson1) [inline README.md:8]: Since these are all part of `Work Completed` I'd suggest changing the subheadings down one level

File: README.md [hunk 3]
```diff
@@ -1,22 +1,114 @@
...
+
+Diagram site: 
+
+Group DTR: https://docs.google.com/document/d/1rITzHzPoXYcNLyEDX3FQwvB98AbPXOpPaZ0PCDH7Foo/edit
+
+## This project's team members
+
+### Artemy Gibson
+
+Github profile: https://github.com/algibson1
```
Comment (algibson1) [inline README.md:91]: We can format these so that the words `Github profile` contain the link. We should also throw in our LinkedIn profiles :)

**Review by algibson1 — APPROVED** [review]

**Review by pcbennett108 — APPROVED** [review]
A solid Readme

**Pull request merged**